src/main/java/org/terasology/ligthandshadow/componentsystem/components/PlayerStatisticsComponent.java

```diff
@@ -34,4 +34,12 @@ public  PlayerStatisticsComponent() {
         this.kills = 0;
         this.deaths = 0;
     }
+
+    public void setKills(int kills) {
```

Comment:
You generally do not need setters or getters in components, since the fields tend to be public you normally just set them directly :-)

Reply:
ok